app/src/main/java/it/niedermann/owncloud/notes/android/appwidget/SingleNoteWidgetFactory.java

```diff
@@ -6,17 +6,25 @@
 import android.content.SharedPreferences;
 import android.os.Bundle;
 import android.preference.PreferenceManager;
+import android.text.Spannable;
+import android.text.SpannableString;
+import android.text.style.StyleSpan;
```

Comment:
These three imports seem to be unused and should be removed.

Reply:
Fixed in 78580addf1da55dea7f2215de57f0322df5a6566